helmfile/charts/grafana-ops/files/welcome.md

```diff
@@ -24,10 +24,9 @@ In case you get lost, don't forget to check out the [public docs](https://elasti
 
 As a user of Compliant Kubernetes you can request to see your Cluster configuration (without secrets) by asking your administrator.
 
-## Support
-
-Did not find the answers you were looking for in the docs? Please contact us via [Jira Service Desk](https://elastisys.atlassian.net/servicedesk/customer/portals)!
-
-Phone: 010-405 42 12
+{{ if .Values.dashboard.extraTextGrafana }}
+## {{ .Values.dashboard.extraTextGrafana }}
+{{ else }}
+{{ end }}
```

Comment:
Can we move the `##` into the value instead? Or why would we want them here?
```suggestion
{{ if .Values.dashboard.extraTextGrafana }}
{{ .Values.dashboard.extraTextGrafana }}
{{ else }}
{{ end }}
```

Reply:
I wasn't able to use the character `#` in when converting from the configs into the `.md` file, wasn't able to escape it either. So right now the first string (until newline) you write in the config will be a header.

Reply:
Ok, and it didn't help to quote the string with `' '` instead of `" "`?
Otherwise I don't think I have any suggestions right now. Could you then in the comment in the config write that the first line will be a header and that you cannot add more?

Reply:
Yup the `#` just comments out the whole string and if I escape it then the `#` character just shows up as text, single quotes didnt help.

I got a suggestion from @robinAwallace  to try to append another `.md` file to the end of this but if I cant get that working I will add the extra comments that you suggested.